From 962f13294766a269802446e251738713a897c040 Mon Sep 17 00:00:00 2001
From: "Joshua M. Clulow" <jmc@joyent.com>
Date: Tue, 5 Feb 2019 22:15:49 +0000
Subject: [PATCH] MANTA-4098 Muskie test suite should set PATH when running
 tests Reviewed by: Cody Peter Mello <melloc@writev.io> Reviewed by: Jan
 Wyszynski <jan.wyszynski@joyent.com> Approved by: Cody Peter Mello
 <melloc@writev.io>

---
 Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index fe5b943..96f2cf3 100644
--- a/Makefile
+++ b/Makefile
@@ -64,7 +64,8 @@ manta-scripts: deps/manta-scripts/.git
 
 .PHONY: test
 test: $(STAMP_NODE_MODULES)
-	$(NODE) ./node_modules/.bin/nodeunit --reporter=tap \
+	PATH=$(ROOT)/$(NODE_INSTALL)/bin:$(PATH) \
+	    $(NODE) ./node_modules/.bin/nodeunit --reporter=tap \
 	    test/*.test.js test/mpu/*.test.js
 
 .PHONY: release
-- 
2.21.0

